src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -653,6 +654,16 @@ export class VegaParser {
         this._setDefaultValue(defaultColor, 'config', 'trail', 'fill');
       }
     }
+
+    // provide right colors for light and dark themes
+    this._setDefaultValue(euiThemeVars.euiColorDarkestShade, 'config', 'title', 'color');
+    this._setDefaultValue(euiThemeVars.euiColorDarkShade, 'config', 'style', 'guide-label', 'fill');
+    this._setDefaultValue(euiThemeVars.euiColorDarkShade, 'config', 'style', 'guide-title', 'fill');
+    this._setDefaultValue(euiThemeVars.euiColorDarkShade, 'config', 'style', 'group-title', 'fill');
```

Comment:
```suggestion
    this._setDefaultValue(euiThemeVars.euiColorDarkestShade, 'config', 'style', 'guide-title', 'fill');
    this._setDefaultValue(euiThemeVars.euiColorDarkestShade, 'config', 'style', 'group-subtitle', 'fill');
    this._setDefaultValue(euiThemeVars.euiColorDarkestShade, 'config', 'style', 'group-title', 'fill');
```
We should threat both the `guide-title` (the axis title) and the `group-subtitle` (used when the user wants a subtitle) with the same color as specified by EUI theme.

Reply:
you can test the subtitle with this spec on vega:
```
"title": {
  "text": "Title Text",
  "subtitle": "Subtitle Text"
}
```

Reply:
done

Reply:
@VladLasitsa I think there you haven't fixed my suggestion to use `euiColorDarkestShade` (as in the current EUI theme) for all of the following styles: `guide-title`, `group-subtitle` and `group-title`.